cowprotocol/accounting/fees/protocol_fees_by_order_class_3703312.sql

```diff
@@ -0,0 +1,44 @@
+with
+protocol_fees_collected as (
+    select --noqa: ST06
+        order_uid, --noqa: RF03
+        cast(cast(data.protocol_fee as varchar) as int256) * data.protocol_fee_native_price / pow(10, 18) as protocol_fee_in_eth, --noqa: RF01, RF03
+        cast(cast(data.protocol_fee as varchar) as int256) as protocol_fee_in_surplus_token, --noqa: RF01
+        case
+            when data.partner_fee_recipient is not null then cast(data.partner_fee as int256) * data.protocol_fee_native_price / pow(10, 18) --noqa: RF01
+        end as partner_fee_eth,
+        cast(cast(data.protocol_fee as varchar) as int256) * data.protocol_fee_native_price / pow(10, 18) - coalesce(case when data.partner_fee_recipient is not null then cast(data.partner_fee as int256) * data.protocol_fee_native_price / pow(10, 18) end, 0) as net_protocol_fee_in_eth, --noqa: RF01
+        data.protocol_fee_token as surplus_token, --noqa: RF01
+        data.quote_gas_cost, --noqa: RF01
+        data.quote_sell_token_price, --noqa: RF01
+        data.quote_sell_amount, --noqa: RF01
+        data.quote_buy_amount, --noqa: RF01
+        tx_hash, --noqa: RF03
+        data.protocol_fee_kind --noqa: RF01
+    from cowswap.raw_order_rewards
+    where block_number > 19068880 and order_uid not in (select cast(order_uid as varchar) from query_3639473) --noqa: RF03
```

Comment:
Please comment what this magic block number is. Also, can we add query 3639473 under version control?